pravega/src/test/scala/docs/scaladsl/PravegaSourceSpec.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.scaladsl
+
+import akka.stream.alpakka.pravega.ReaderSettings
+import akka.stream.alpakka.pravega.scaladsl.Pravega
+import akka.stream.scaladsl.{Keep, Sink}
+
+import io.pravega.client.stream.impl.JavaSerializer
+import io.pravega.client.stream.EventWriterConfig
+import akka.stream.KillSwitches
+
+class PravegaSourceSpec extends PravegaBaseSpec {
+
+  final implicit val eventWriterConfig = EventWriterConfig.builder().build()
+
+  implicit val serializer = new JavaSerializer[String]
+  implicit val settings = new ReaderSettings(groupName, readerId, serializer)
+
+  "Pravega connector" should {
+    "compiles settings" in {
+      new ReaderSettings(groupName, readerId, serializer)
+    }
+
+    "compiles sources" in {
+
+      Pravega.source(scope, streamName)
+    }
+
+    "runs sources" in {
```

Comment:
Is anything actually read from Pravega in this test?  I don't see anything where data is setup first.

Reply:
Is it just to test the control/kill switch?  Maybe the test should be renamed accordingly.

Reply:
Nevermind, I see the read in the logs now.  I assume this is the data written by `PravegaSinkSpec`?  They should probably be decoupled.

Reply:
Right, test need to be cleaned up ...